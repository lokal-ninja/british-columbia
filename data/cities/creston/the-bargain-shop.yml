id: 219619146
type: node
latitude: 49.09544
longitude: -116.5116
postcode: V0B 1G4
city: Creston
street: Canyon Street
housenumber: "1119"
phone: ""
opening_hours: ""
website: ""
