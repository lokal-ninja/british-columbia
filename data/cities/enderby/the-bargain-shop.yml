id: 10076149720
type: node
latitude: 50.55043
longitude: -119.13794
postcode: ""
city: Enderby
street: "Cliff Avenue"
housenumber: 
phone: ""
opening_hours: ""
website: ""