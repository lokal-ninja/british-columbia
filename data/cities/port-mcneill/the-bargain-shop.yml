id: 7097250912
type: node
latitude: 50.587439
longitude: -127.08415670000001
postcode: "V0N 2R0"
city: Port McNeill
street: "Hemlock Street"
housenumber: 311
phone: ""
opening_hours: ""
website: ""
