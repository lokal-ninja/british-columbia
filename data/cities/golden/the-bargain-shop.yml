id: 516881663
type: way
latitude: 51.302319600000004
longitude: -116.96899320000001
postcode: ""
city: Golden
street: ""
housenumber: 
phone: ""
opening_hours: "Mo-Sa 09:00-21:00; Su 10:00-18:00"
website: "http://www.thebargainshop.com"
